src/win/pty.cc

```diff
@@ -311,18 +313,21 @@ static NAN_METHOD(PtyResize) {
 static NAN_METHOD(PtyKill) {
   Nan::HandleScope scope;
 
-  if (info.Length() != 1
-    || !info[0]->IsNumber()) // pid
+  if (info.Length() != 2
+    || !info[0]->IsNumber() // pid
+    || !info[1]->IsNumber()) // innerPidHandle
   {
     return Nan::ThrowError("Usage: pty.kill(pid)");
```

Comment:
nit: I suppose this usage string is now incorrect -- i.e. it should be something like `pty.kill(pid, innerPidHandle)`